SU2_CFD/src/solvers/CTurbSASolver.cpp

```diff
@@ -1591,3 +1588,54 @@ void CTurbSASolver::SetUniformInlet(const CConfig* config, unsigned short iMarke
   }
 
 }
+
+void CTurbSASolver::ComputeUnderRelaxationFactor(const CConfig *config) {
+
+  /* Apply the turbulent under-relaxation to the SA variants. The
+   SA_NEG model is more robust due to allowing for negative nu_tilde,
+   so the under-relaxation is not applied to that variant. */
+
+  bool sa_model = ((config->GetKind_Turb_Model() == SA)        ||
+                   (config->GetKind_Turb_Model() == SA_E)      ||
+                   (config->GetKind_Turb_Model() == SA_COMP)   ||
+                   (config->GetKind_Turb_Model() == SA_E_COMP));
```

Comment:
```suggestion
  if (config->GetKind_Turb_Model() == SA_NEG) return;
```

Reply:
Although I am not fully sold on reversing these kinds of logics, who knows what is about to come